CONTINUITY.md

```
@@ -1,62 +1,64 @@
-Commit de los cambios actuales → push → restore en producción (next.actiglobal.com)
+Commit aprobado → push → PR.
 
 Criterio de avance:
-Commit creado, push hecho, backup de actiglobal-restore.dev disponible para restore.
+PR abierto, CI verde.
 
 ---
 
 ## Estado actual
 
 ### Ya cerrado
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)